hooks/conan-center.py

```diff
@@ -724,6 +725,31 @@ def test(out):
                     out.warn("The topic '{}' is invalid; even names and acronyms should be formatted "
                              "entirely in lowercase.".format(topic))
 
+    @run_test("KB-H065", output)
+    def test(out):
+        def _find_required_conan_version(conanfile_content):
+            match = re.search(r"^\s*required_conan_version\s*=\s*\">=\s*([\d\.]+)\"",
```

Comment:
```suggestion
            match = re.search(r"^\s*required_conan_version\s*=\s*\["']>=\s*([\d\.]+)\["']",
```

single quote are allowed too.